global-script.lua

```diff
@@ -3079,8 +3087,30 @@ function tCompare(t1,t2)
         newTab = {}
         for i,v in ipairs(tab) do
             newTab[i] = table.concat(v,",")
-    end
-    return table.concat(newTab,"|")
+        end
+        return table.concat(newTab,"|")
     end
     if cc2(t1) == cc2(t2) then return true else return false end
-end
\ No newline at end of file
+end
+function togglePlayer(player, color)
+    player = Player[player.color]
+    player.changeColor(color)
+    Wait.frames(function ()
+        Player[color].lookAt({
+            position = getObjectFromGUID(playerCounters[color]).getPosition() - Vector(0,0,15),
+            pitch = 70,
+            distance = 30,
+        })
+    end, 2)
+end
+function onPlayerChangeColor(color)
+    for color, counterGUID in pairs(playerCounters) do
+        counter = getObjectFromGUID(counterGUID)
+        if Player[color].seated then
```

Comment:
you could also check to make sure that the seat has a spirit picked something along the lines of `#getObjectFromGUID(PlayerBags[color]).getObjects() == 0`

Reply:
so on second thought maybe you have the buttons available if the game hasn't started, but then once the game has started all buttons for non seats without spirits go away?

Reply:
I'd rather not be that prescriptive. I'm fairly certain that I've selected spirits after start the game, at times.

Reply:
alright so here's my thinking. before the game has started the button will show for all empty seats. but once the game has started the button only shows for empty seats which contain a spirit

Reply:
My point is that I sometimes start the game before picking spirits, so I want to switch to seats without a spirit, to be able to pick them. I definitely understand the reasoning for wanting to remove them, but I think being less prescriptive makes sense here.

Reply:
you can still switch seats the old fashioned way, just not a fan of cluttering up that space with buttons forever sticking around. just made a change last night to remove all the text and ready tokens until after a spirit has been picked

Reply:
I think the flexibility is worth a (tiny) bit of clutter somewhere that most people aren't going to be paying attention to anyway (I similarly am not convinced removing the energy tracker makes sense). For example, I'm not sure why someone who wants to play against a randomized adversary/scenario but choose spirits afterward should be punished by having to use a slightly more cumbersome way to switch between colors to pick spirits.

In any case, I will definitely patch the current behavior in locally, though I'd prefer not to have to do that.